src/main/rx/jetiexbus.c

```diff
@@ -230,8 +229,7 @@ static uint8_t jetiExBusFrameStatus(rxRuntimeState_t *rxRuntimeState)
         if (jetiExBusCalcCRC16(jetiExBusChannelFrame, jetiExBusChannelFrame[EXBUS_HEADER_MSG_LEN]) == 0) {
             jetiExBusDecodeChannelFrame(jetiExBusChannelFrame);
             frameStatus = RX_FRAME_COMPLETE;
-        } else {
-            lastRcFrameTimeUs = 0;  // We received a frame but it wasn't valid new channel data
+            lastRcFrameTimeUs = jetiTimeStampRequest;
```

Comment:
I'm not sure about this. The point of recording the rc channels frame interval is to base it on a valid packet, not simply receiving it. If the packets have crc errors they shouldn't be considered valid from a timing perspective.

Reply:
Well, this is within the `if (jetiExBusCalcCRC16())`, so this should be taken care of. Or am I overlooking something?

Reply:
Sorry - I missed that the `} else {` was removed. Thought it was still in crc failed section.